common/features/org.jboss.tools.common.all.feature/feature.properties

```diff
@@ -1,9 +1,6 @@
 featureName=JBoss Tools Common All Feature
 featureProvider=JBoss by Red Hat
 
-# "updateSiteName" property - label for the update site
-updateSiteName=JBossTools Update Site
```

Comment:
Not related. But was updating feature.properties and saw these unreferenced properties so figured I'd remove them too. They're useless clutter.